### PR TITLE
Feature/sol script

### PR DESCRIPTION
**Pull request by Sotatek-KhanhLe**

### CODE REVIEW

**Comment by Sotatek-KhanhLe** [comment]
I think with case smart contract we could use only getArgumentAll to take all argument and then sign for once, since solana smart contract transaction data is vast and kind of unpredictable to target on specific length for each fields. In Addition, with other coin I see you not showing any transaction information on card with smart contract cases therefore this solution kind of acceptable

**Comment by hank121314** [comment]
> I think with case smart contract we could use only getArgumentAll to take all argument and then sign for once, since solana smart contract transaction data is vast and kind of unpredictable to target on specific length for each fields. In Addition, with other coin I see you not showing any transaction information on card with smart contract cases, and solana signing process is pretty strange forward no need to complicate encoding for each fields of argument upload to card. therefore I think this solution is kind of acceptable

Since smart contract does not need to show the data, using `getArgumentAll` to `data` field is acceptable.

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,6 @@
 **/nbdist/
 **/.nb-gradle/
 **/node_modules/
+**/.idea
 /bin/
+*.iml
```
Comment (hank121314) [inline .gitignore:12]: Unrelated changes.

File: script-generator/.gitignore [hunk 2]
```diff
@@ -6,10 +6,9 @@
...
 **/.nb-gradle/
+**/.idea/
 /bin/
 .settings/
-<<<<<<< HEAD
-=======
 .classpath
 .project
->>>>>>> dev
+*.iml
```
Comment (hank121314) [inline script-generator/.gitignore:14]: Unrelated changes.

**Review by hank121314 — CHANGES_REQUESTED** [review]

File: script-generator/.gitignore [hunk 3]
```diff
@@ -8,8 +8,5 @@
 **/.nb-gradle/
 /bin/
 .settings/
-<<<<<<< HEAD
-=======
 .classpath
 .project
->>>>>>> dev
```
Comment (hank121314) [inline script-generator/.gitignore:15]: Please remove these changes.

**Review by hank121314 — CHANGES_REQUESTED** [review]

File: script-generator/.gitignore [hunk 4]
```diff
@@ -12,4 +13,4 @@
 =======
 .classpath
 .project
->>>>>>> dev
+>>>>>>> dev
```
Comment (hank121314) [inline script-generator/.gitignore:16]: Please remove these changes.

**Review by hank121314 — CHANGES_REQUESTED** [review]

File: script-generator/.gitignore (hunk 4 above)
Comment (Sotatek-KhanhLe) [reply script-generator/.gitignore:15]: @hank121314 I have removed unrelated change you mention, please check it

Comment (hank121314) [reply script-generator/.gitignore:15]: Seems still exist in the latest commit.

**Review by hank121314 — CHANGES_REQUESTED** [review]

**Review by hank121314 — APPROVED** [review]

**Review by srt1678 — APPROVED** [review]
